recipes/bohra/meta.yaml

```diff
@@ -0,0 +1,86 @@
+{% set name = "bohra" %}
+{% set version = "1.0.19" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "f99205fbdad0a1822a0eb3ae72f55d76b1fcd9d03cbe60923b9d74849929e76b"
+
+build:
+  noarch: generic
+  number: 0
+  entry_points:
+    - bohra=bohra.bohra:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - biopython >=1.70
+    - jinja2
+    - numpy
+    - packaging
+    - pandas >=0.23.0
+    - pip
+    - psutil
+    - pytest-runner
+    - python >=3.6
+    - pytest
+    - sh
+    - snakemake >=5.4.0
+    - svgwrite
+    - snippy
+    - shovill
+    - roary
+    - prokka
+    - kraken2
+    - abricate
+    - mlst
+    - iqtree
+    - seqtk
+    - snp-dists
+    - zlib
+
+  run:
+    - biopython >=1.70
+    - jinja2
+    - numpy
+    - packaging
+    - pandas >=0.23.0
+    - psutil
+    - pytest-runner
+    - pytest
+    - python
+    - sh
+    - snakemake >=5.4.0
+    - svgwrite
+    - snippy
+    - shovill
+    - roary
+    - prokka
+    - kraken2
+    - abricate
+    - mlst
+    - iqtree
+    - seqtk
+    - snp-dists
+    - zlib
```

Comment:
why is zlib here needed?

Reply:
ping

Reply:
@bgruening the zlib has now been removed from current recipe